README.md

```diff
@@ -18,6 +18,25 @@ Those packages are built for:
 
 ## Installation
 
+### Repository key
+
+Packages provided by the repository are signed. In order to verify the integrity of the packages, you need to import the public key. First, verify that the fingerprint of `key.asc` matches `4EFC 5906 96CB 15B8 7C73  A3AD 82CC 8797 C838 DCFD`:
+
+```sh
+curl -fsSL https://pkgs.zabbly.com/key.asc | gpg --show-keys --fingerprint
```

Comment:
I think we should still keep a link to `key.asc` so users can click that and copy/paste should they prefer it.

Reply:
How about this? a38e8c6373e78689fbe4a5990ede3ca15efbd8c6